src/components/CommentCard.jsx

```diff
@@ -1,11 +1,34 @@
-function CommentCard({ comment }) {
+import { useContext } from "react";
+import UserContext from "../contexts/User";
+import { deleteComment } from "../../utils/api";
+
+function CommentCard({ comment, setComments}) {
+  const { loggedInUser } = useContext(UserContext);
+
+  function handleDelete() {
+    deleteComment(comment.comment_id).then(() => {
```

Comment:
We could feedback to the user when the request is not successful, for example when is made while offline 🙂